### ISSUE TITLE
Unsupported beautification result 'Parsing error

### ISSUE THREAD

**Issue by BMOv2600** [Glavin001/atom-beautify#1547]
Beautify cannot parse some CSS. This error also prevents beautify from beautifying any CSS in the file.

It dislikes the following:

```
@media(min-width:480px) and (max-width:619px) {
    ul.footer-menu-list li: nth-child(3) {
    margin-right: 0;
}
ul.footer-menu-list li:first-child {
    margin-left: 0;
}
.footer-menu-list-item {
    margin: 20px 16px;
}
}
```

This is what the code looked like before:

```
@media(min-width:480px) and (max-width:619px) {
    ul.footer-menu-list li: nth-child(3) {
    margin-right: 0;
}
ul.footer-menu-list li:first-child {
    margin-left: 0;
}
.footer-menu-list-item {
    margin: 20px 16px;
}
}
```

# Expected Output

The beautified code should have looked like this:

```
@media(min-width:480px) and (max-width:619px) 
	{
    	ul.footer-menu-list li: nth-child(3) 
      {
        margin-right: 0;
      }
    ul.footer-menu-list li:first-child 
      {
        margin-left: 0;
      }
    .footer-menu-list-item 
      {
        margin: 20px 16px;
      }
    }
```

# Actual Output

The beautified code actually looked like this:

```
@media(min-width:480px) and (max-width:619px) {
    ul.footer-menu-list li: nth-child(3) {
    margin-right: 0;
}
ul.footer-menu-list li:first-child {
    margin-left: 0;
}
.footer-menu-list-item {
    margin: 20px 16px;
}
}
```

# Steps to Reproduce

Error: Unsupported beautification result 'Parsing error: Please check the validity of the block starting from line #1616
1614 | }
1615 | @media(min-width:480px) and (max-width:619px) {
1616*|     ul.footer-menu-list li: nth-child(3) {
1617 |     margin-right: 0;
1618 | }
Gonzales PE version: 3.0.0-28
Syntax: css'.
    at beautifyCompleted (/Users/deauberts/.atom/packages/atom-beautify/src/beautify.coffee:135:21)
    at tryCatcher (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:510:31)
    at Promise._settlePromise (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:567:18)
    at Promise._settlePromise0 (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:612:10)
    at Promise._settlePromises (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/promise.js:687:18)
    at Async._drainQueue (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:143:10)
    at Async.drainQueues (/Users/deauberts/.atom/packages/atom-beautify/node_modules/bluebird/js/release/async.js:17:14)

# Debug

Here is a link to the `debug.md` Gist:
[https://gist.github.com/BMOv2600/acc5abfd3b26ef8a042674ba4b33bd3d](https://gist.github.com/BMOv2600/acc5abfd3b26ef8a042674ba4b33bd3d)

# Checklist

I have:
- [ X] Tried uninstalling and reinstalling Atom Beautify to ensure it installed properly
- [ X] Reloaded (or restarted) Atom to ensure it is not a caching issue
- [ X] Searched through existing Atom Beautify Issues at https://github.com/Glavin001/atom-beautify/issues
so I know this is not a duplicate issue
- [ X] Filled out the Input, Expected, and Actual sections above or have edited/removed them in a way that fully describes the issue.
- [ X] Generated debugging information by executing `Atom Beautify: Help Debug Editor` command in Atom and added link for `debug.md` Gist to this issue

**Comment by tonyganch** [comment]
Hi guys!
First of all, `li: nth-child(3)` is not valid css, there should be no space after `:`, that's why you get parsing error.
Also, Atom Beautify uses outdated version of csscomb package (3.1 instead of 4.0), I would suggest you update to the latest one, it has a lot of parsing issues fixed.

**Comment by BMOv2600** [comment]
Sorry. I used an unminifier that might have added the space. Glad you caught that.

How do I update Beautify's csscomb package?

**Comment by Glavin001** [comment]
https://github.com/Glavin001/atom-beautify/pull/1660 has been merged.

Published to v0.29.26
